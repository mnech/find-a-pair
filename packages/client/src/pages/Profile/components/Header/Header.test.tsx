import React from 'react';
import { render } from '@testing-library/react';
import { Header } from './index';

describe('Header profile component', () => {
  describe('should be rendered correctly Header component', () => {
    test('default', () => {
      const testUser = {
        first_name: 'Иван',
        second_name: 'Иванов',
        display_name: 'Иванушка',
        login: 'ivanovv',
        email: 'ivanov@yandex.ru',
        phone: '89999999999',
        avatar: '',
      };

      const { asFragment } = render(
        <Header avatar={testUser.avatar} name={testUser.first_name} />,
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
