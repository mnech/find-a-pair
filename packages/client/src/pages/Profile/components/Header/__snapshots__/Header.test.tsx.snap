// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header profile component should be rendered correctly Header component default 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <img
      alt="Аватар"
      class="avatar rounded-3 shadow"
      src=""
    />
    <input
      accept="image/*"
      hidden=""
      type="file"
    />
    <h1>
      Иван
    </h1>
  </header>
</DocumentFragment>
`;
